pkg/secrets/censor.go

```diff
@@ -31,3 +34,23 @@ func (c *DynamicCensor) AddSecrets(s ...string) {
 	c.secrets.Insert(s...)
 	c.ReloadingCensorer.Refresh(c.secrets.List()...)
 }
+
+// ReadFromEnv loads content from the environment and adds it to the list.
+func (c *DynamicCensor) ReadFromEnv(name string) string {
+	ret := os.Getenv(name)
+	if ret != "" {
+		c.AddSecrets(ret)
+	}
+	return ret
+}
+
+// ReadFromFile loads content from a file and adds it to the list.
+func (c *DynamicCensor) ReadFromFile(path string) (string, error) {
+	bytes, err := ioutil.ReadFile(path)
+	if err != nil {
+		return "", err
+	}
+	ret := strings.TrimSpace(string(bytes))
+	c.AddSecrets(ret)
```

Comment:
Should we also guard against an empty string (a file that contains an empty string or just whitespace...). I remember that empty string as a censored secret caused mayhem in the past (perhaps `AddSecrets` should guard agaist it? but then the check in `ReadFromEnv` is useless)

Reply:
No, the censorer handles this.